integrations/custom/http_server/client.py

```diff
@@ -33,21 +32,33 @@ def __init__(
         timeout: int = 30,
         verify_ssl: bool = True,
         max_concurrent_requests: int = 10,
+        custom_headers: Optional[Dict[str, str]] = None,
     ):
         self.base_url = base_url.rstrip("/")
         self.auth_type = auth_type
         self.auth_config = auth_config
         self.pagination_config = pagination_config
         self.timeout = timeout
         self.verify_ssl = verify_ssl
+        self.custom_headers = custom_headers or {}
 
         # Use Ocean's built-in HTTP client with retry and rate limiting
-        self.client = http_async_client
+        # If SSL verification is disabled, create a custom client
+        if not verify_ssl:
+            logger.warning(
+                "SSL verification is disabled - not recommended for production"
+            )
+            self.client = httpx.AsyncClient(verify=False)
```

Comment:
```suggestion
            self.client = OceanAsyncClient(
                RetryTransport,
                timeout=ocean.config.client_timeout,
                verify=False
            )
```
that way u don't lose the retries and timeout configs